stingray/tests/test_powerspectrum.py

```diff
@@ -424,61 +424,122 @@ def test_compute_rms_wrong_norm(self):
         # This will now pass, due to changes on 2022-10-10
         ps.compute_rms(0, 10)
 
-    def test_fractional_rms_in_frac_norm_is_consistent(self):
+    def test_compute_rms_rebinning_is_consistent(self):
         time = np.arange(0, 100, 1) + 0.5
 
         poisson_counts = np.random.poisson(100.0,
-                                           size=time.shape[0])
+                                        size=time.shape[0])
 
         lc = Lightcurve(time, counts=poisson_counts, dt=1,
                         gti=[[0, 100]])
         ps = Powerspectrum(lc, norm="leahy")
-        rms_ps_l, rms_err_l = ps.compute_rms(min_freq=ps.freq[1],
-                                             max_freq=ps.freq[-1],
-                                             white_noise_offset=0)
+        ps_rebinned = ps.rebin_log()
+        rms, err_rms = ps.compute_rms(min_freq=ps.freq[1], max_freq=ps.freq[-2],
+            poisson_noise_level=0)
+        rms_reb, err_rms_reb = ps_rebinned.compute_rms(min_freq=ps.freq[1], 
+        max_freq=ps.freq[-2], poisson_noise_level=0)
+        assert np.isclose(rms, rms_reb, atol=0.01, rtol=0.01)
 
-        ps = Powerspectrum(lc, norm="frac")
-        rms_ps, rms_err = ps.compute_rms(min_freq=ps.freq[1],
-                                         max_freq=ps.freq[-1],
-                                         white_noise_offset=0)
-        assert np.allclose(rms_ps, rms_ps_l, atol=0.01)
-        assert np.allclose(rms_err, rms_err_l, atol=0.01)
-
-    def test_fractional_rms_in_frac_norm_is_consistent_averaged(self):
-        time = np.arange(0, 400, 1) + 0.5
 
-        poisson_counts = np.random.poisson(100.0,
-                                           size=time.shape[0])
+    def test_fractional_rms_in_frac_norm_is_consistent(self):
+            time = np.arange(0, 100, 1) + 0.5
```

Comment:
Minor comment: this is badly formatted

Reply:
Hi @matteobachetti, sorry I don't understand it. Which line is badly formatted?

Reply:
Four spaces each indent ;)